### PR TITLE
Changed setup section to markdown

### PR DESCRIPTION
**Pull request by parkand1**

### CODE REVIEW

File: Documentation/setup.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# EKS Factory Reference Guides
```
Comment (kcoleman731) [inline Documentation/setup.md:1]: @parkand1 looks like the majority of this overlaps with the existing readme. Do you want to add any updates/differences to the existing readme (so we don't have duplicate stuff for the time being)?

**Review by kcoleman731 — COMMENTED** [review]
@parkand1 looks like @shapirov103 branch has a /docs dir already. Should we use that? You may need to rebase.

File: Documentation/setup.md (hunk 1 above)
Comment (shapirov103) [reply Documentation/setup.md:1]: +1, let's move pertinent parts to readme.

File: Documentation/setup.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        new CdkEksBlueprintStack(app, {id: 'east-test-1', addOns: addOns, teams: []}, {
+          env: {
+              account: 'AWS_ACCOUNT_ID',
+              region: 'REGION'
+          },
+        });
+ ```       
+
+## Deploy the stack
+
```
Comment (shapirov103) [inline Documentation/setup.md:66]: I mentioned elsewhere: this is were we need to add `cdk run build`.

File: Documentation/setup.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+1. Run the following to deploy your stack: cdk deploy
+2. Once the deployment is complete you should see the following output 
+
+```
+✅ east-test-1
+
+Outputs:
+east-test-1.easttest1ClusterName8D8E5E5E = east-test-1
+east-test-1.easttest1ConfigCommand25ABB520 = aws eks update-kubeconfig --name east-test-1 --region us-east-2 —role-arn arn:aws:iam::572179936737:role/east-test-1-easttest1MastersRole19C9E55A-EESH5L8KOXMJ
```
Comment (shapirov103) [inline Documentation/setup.md:75]: let's obfuscate the isengard account number.

**Review by shapirov103 — CHANGES_REQUESTED** [review]
1. Let's move the guide (public facing, only relevant details, not revealing any accounts, etc.) under docs. 
2. Let's integrate relevant parts to the README and cross-reference.
3. General cleanup and I am waiting for argo bootstrapping portion as well.

File: Documentation/setup.md (hunk 1 above)
Comment (parkand1) [reply Documentation/setup.md:1]: Sounds good, will do

**Comment by parkand1** [comment]
I accidentally deleted my forked repo so i'm going to just branch off the original repo and make my changes there.

**Pull request closed without merging**